.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # General ownership
-* @gesundheitscloud/native-app-core
+* @gesundheitscloud/native-app-core @d4l-data4life/phdc-core
```

Comment:
@wmontwe is `@gesundheitscloud/native-app-core` still needed?

Reply:
It is not

Reply:
@alessioborraccino Could please then remove it?